cibuildwheel/util.py

```diff
@@ -55,8 +55,8 @@ def read_python_configs(config: PlatformName) -> List[Dict[str, str]]:
     return results
 
 
-class BuildSelector:
-    def __init__(self, build_config: str, skip_config: str):
+class IndentifierSelector:
```

Comment:
Typo- `IdentifierSelector`. But otherwise this design works for me :)

Reply:
I had to stare at this for a minute to figure out what was different. 😳